code/modules/mob/living/carbon/xenomorph/xenomorph.dm

```diff
@@ -55,7 +55,7 @@
 
 	RegisterSignal(src, COMSIG_GRAB_SELF_ATTACK, .proc/devour_grabbed) //Devour ability.
 
-	AddComponent(/datum/component/bump_attack)
+	AddComponent(/datum/component/directional_attack)
```

Comment:
I still think we should keep bump, and just add directional to it.

Reply:
disagree, directional should replace bump

Reply:
balance is kinda bad with bump still in, directional is less severe a factor compared to bump

Reply:
I don't see bump as a balance concern though, just another way to "click" someone. WASD is easier than clicking for sure, but bump doesn't give them more damage or free hits, it just makes it that when a xeno does get in range to attack they are more likely too because controls are not inhibiting them.